src/screens/Podcast/components/EpisodeItem/episodeItem.tsx

```diff
@@ -1,27 +1,22 @@
 import React from 'react'
 import { View, TouchableOpacity } from 'react-native'
+import { Episode } from 'src/common/types'
+import { getFormattedDate, DateFormatType } from 'src/helpers'
 import PlayIcon from 'src/assets/images/playEpisode.svg'
 import { Heading, HeadingType, PlainText } from 'src/components'
 import styles from './styles'
 
-type Episode = {
-  name: string
-  createdAt: string
-}
-
 type Props = {
-  number: number
+  episodePosition: number
```

Comment:
This component is related to Episode. We shouldn't use part 'episode' here.

Reply:
Why? In this file we use many strings with prefix "episode". I can rename "episodePosition" in "episodeItemPosition" for example. But if i name this to simple - "position", as for me it will be not clear, what kind of position mean.